test/parallel/test-async-hooks-enable-recursive.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const common = require('../common');
+const async_hooks = require('async_hooks');
+const crypto = require('crypto');
```

Comment:
if you're using `crypto` you need:
```
if (!common.hasCrypto)
  common.skip('missing crypto');
```
AFAICT this should work with `fs` or `stream`...